.vscode/settings.json

```
@@ -0,0 +1,17 @@
+{
+    "editor.tabSize": 2,
+    "files.exclude": {
+        "**/.dart_tool": true,
+        "**/Icon?": true
+    },
+    "files.trimFinalNewlines": true,
+    "files.trimTrailingWhitespace": true,
+    "files.insertFinalNewline": true,
+    "html.format.endWithNewline": true,
+    "dart.lineLength": 120,
+    "[dart]": {
+        "editor.rulers": [
+            120
```

I just had a chat with Dan and we were discussing 160 for the backend code - I'd be inclined to revisit our 120 and use the same number here as well? what do you think?
```suggestion
            160
```